scripts/packageManifestV2Extensions.js

```diff
@@ -0,0 +1,150 @@
+import fs from 'fs'
+import unzip from 'unzip-crx-3'
+import path from 'path'
+import crypto from 'crypto'
+import commander from 'commander'
+import util from '../lib/util.js'
+import crx from '../lib/crx.js'
+
+const verifyChecksum = (data, sha512) => {
+  return sha512 === crypto.createHash('sha512').update(data).digest('hex')
+}
+
+const downloadExtension = async (manifest) => {
+  const buildPath = path.join('build', manifest.name)
+  const download = path.join(buildPath, 'download')
+  const unpacked = path.join(buildPath, 'unpacked')
+
+  fs.mkdirSync(download, { recursive: true })
+  fs.mkdirSync(unpacked, { recursive: true })
+
+  const response = await fetch(manifest.url)
+  const data = Buffer.from(await response.arrayBuffer())
+  if (!verifyChecksum(data, manifest.sha512)) {
+    console.error(`${manifest.name} checksum verification failed`)
+    process.exit(1)
```

Comment:
hm, why `exit(1)` instead of `throw new Error(..)`?

Reply:
Spied in other files

Reply:
But, `installErrorHandlers` do what we want, it catches all uncaught exceptions prints it in `error` and `exit(1)`